### DISCUSSION TITLE
V7: Multiple Dependent Field Validation On Change

### DISCUSSION THREAD

**Discussion by kensukesaito** [react-hook-form/react-hook-form#4618] [Q&A] [answered]
I posted following issue yesterday, got replies from @leapful and @bluebill1049 (thanks!), but still doesn't work.

v7: Does not work <input {...register('example')} onChange={...}>
https://github.com/react-hook-form/react-hook-form/issues/4616

Have a look at the following codes.
https://codesandbox.io/s/romantic-dijkstra-7qcp5

I would like to validate `from` <= `to` in real time.

It looks working well, but there is inexplicable behavior.

Try:
1. from: 1
2. to: 2
3. from: 3 (shows an error because `from` is greater than `to`: expected)
4. to: 4 (still shows an error although `from` is less than `to`: unexpected)

@leapful suggested using `useForm({ reValidateMode: 'onChange' })` , but doesn't affect.

Does anyone have any ideas?

/src/registerOutFromRender.tsx:
I tried @bluebill1049 's solution, but still doesn't work.

/src/workingExampleForV6.tsx:
This is a working example for V6 (use onChange and trigger()).

**Comment by jorisre** [comment, marked as answer]
Hi @kensukesaito 👋🏻 

Here is a possible solution: https://codesandbox.io/s/restless-morning-4fddl?file=/src/App.tsx

The idea is to trigger manually the validation of the `from` input when `to` change

**Reply by kensukesaito** [reply]
Thank you @jorisre for your great solution!
